part-overview.tex

```diff
@@ -133,7 +133,7 @@ \section{Supported functionality}
 \item Application administration
 \end{itemize}
 
-These add-ons can be installed:
+These \glspl{add-on}  can be installed:

```

Comment:
Ah. You can't know this yet, but budgetting has been integrated into the application, so it's no longer an add-on. I'm not aware of "enhanced AR/ AP" being an option these days (and although I wrote it, I don't remember what it was actually about...)

Reply:
Removed both in the next update.
Thanks